packages/wonder-blocks-testing/src/fetch/fetch-request-matches-mock.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import type {FetchMockOperation} from "./types.js";
+
+/**
+ * Get the URL from the given RequestInfo.
+ *
+ * Since we could be running in Node or in JSDOM, we don't check instance
+ * types, but just use a heuristic so that this works without knowing what
+ * was polyfilling things.
+ */
+const getHref = (input: RequestInfo): string => {
+    if (typeof input === "string") {
+        return input;
+    } else if (typeof input.url === "string") {
+        return input.url;
+    } else if (typeof input.href === "string") {
+        return input.href;
+    } else {
+        throw new Error(`Unsupported input type`);
+    }
+};
+
+/**
+ * Determines if a given fetch invocation matches the given mock.
+ */
+export const fetchRequestMatchesMock = (
```

Comment:
For now, we're just supporting URL matches, but we could envisage matching on header values, method, etc.